### PR TITLE
add coordinates to places items

### PR DESCRIPTION
**Pull request by Inna-r**

### CODE REVIEW

File: bhs_api/item.py [hunk 1]
```diff
@@ -369,3 +368,11 @@ def create_slug(document, collection_name):
     return ret
 
 
+def get_geojson():
```
Comment (daonb) [inline bhs_api/item.py]: The name can be better (it does not return json).
Maybe get_places_geo ?

File: bhs_api/item.py [hunk 2]
```diff
@@ -369,3 +368,11 @@ def create_slug(document, collection_name):
...
+    filters = SHOW_FILTER.copy()
+    filters['geometry'] = {'$exists': True}
+    filters['Header.En'] = {'$nin' : [None, '']}
+    response = []
+    data = {}
```
Comment (daonb) [inline bhs_api/item.py]: 375-376 should be deleted

File: conf/app_server.yaml [hunk 3]
```diff
@@ -34,3 +34,5 @@ elasticsearch_host: bhs-infra
 redis_host: localhost
 redis_port: 6379
 caching_ttl: 3600
+#opencagedata:
+geocoder_key: 0ef3d3f7ec66ed4a1f0ab77bada03cff
```
Comment (daonb) [inline conf/app_server.yaml]: better replace the key with `badkey` so it'll be clear this key can not be used

File: migration/tasks.py [hunk 4]
```diff
@@ -190,6 +191,10 @@ def update_collection(collection, query, doc):
 
 
 def update_doc(collection, document):
+    # update place items with geojson
+    if collection.name = 'places':
+        get_geojson(document)
```
Comment (daonb) [inline migration/tasks.py]: get_geojson does not change `document`. you have to add it's return value to the document

File: scripts/add_coordinates_to_places.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--db',
+                        help='the db to run on defaults to the value in /etc/bhs/config.yml')
+    return parser.parse_args()
+
+
+def get_geojson(doc):
```
Comment (daonb) [inline scripts/add_coordinates_to_places.py]: it does not return json. maybe call it get_place_geo?

**Comment by daonb** [comment]
Much better, but still two issues:

- the names are still wrong: get_place_geo get the geometry for a document from opencage while get_places_geo returns a mongo cursor to all the places that have geo. The names suggest the function do almost the same thing, but in truth, it's very different. I suggest to remove get_places_geo and just add the code directly in v1_endpoints.py
- the name of the script is too long and not descriptive enough. replace `add_coordinates_to_place` with `get_places_geo` (provided you remove the function)

**Comment by nuritgazit** [comment]
@OriHoch @TheGrandVizier are we still using this "opencage" service by OpenCage?

**Comment by OriHoch** [comment]
@nuritgazit we are not using it in the new architecture at the moment

**Pull request merged**